mise.toml

```diff
@@ -25,8 +25,8 @@ go = "1.23.8"
 "go:github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway" = "2.20.0"
 "go:github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2" = "2.27.1"
 "go:github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc" = "1.5.1"
-golangci-lint = "1.61.0"
-gotestsum = "1.10.0" # should be kept in sync with go.mod
+golangci-lint = "2.6.2"
```

Comment:
Should I expect a similar golangci-lint upgrade in rdk soon with a bunch of chaos?